### PR TITLE
Use JWT to authenticate /api/checks calls

### PR DESCRIPTION
**Pull request by fabriziosestito**
Adds missing JWT authentication to checks related API calls in the FE, by simply replacing the axios client.

### CODE REVIEW

**Review by arbulu89 — APPROVED** [review]
Nice trick!
So now it used the client which already has the token.
Let's test before merging hehe

File: assets/js/lib/api/wanda.js [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
 
 const baseURL = process.env.WANDA_URL;
 
-export const wandaClient = axios.create({
-  baseURL,
-});
-
 export const getExecutionResult = (executionID) =>
-  wandaClient.get(`/api/checks/executions/${executionID}`);
+  networkClient.get(`/api/checks/executions/${executionID}`, { baseURL: baseURL });
```
Comment (dottorblaster) [inline assets/js/lib/api/wanda.js:7]: You could declare a default config for all the calls and pass it over to functions or spread when needed

```js
const defaultConfig = { baseURL };

export const getCatalog = (env) =>
  networkClient.get(`/api/checks/catalog`, { ...defaultConfig, params: env });
```

**Review by dottorblaster — CHANGES_REQUESTED** [review]
Just a tiny change then we can merge

**Comment by CDimonaco** [comment]
@dottorblaster Nice catch, but I would choose another name, default config maybe is misleading, the default config for the client is in the network library, but idk

**Comment by dottorblaster** [comment]
@CDimonaco I suck at naming things, I agree :smile:

**Comment by fabriziosestito** [comment]
Closing as duplicate

File: assets/js/lib/api/wanda.js (hunk 1 above)
Comment (fabriziosestito) [reply assets/js/lib/api/wanda.js:7]: @CDimonaco i am going with config since the doc states `axios.get(url[, config])`

**Pull request closed without merging**